JSDemos/Demos/Validation/Overview/description.md

```diff
@@ -1,9 +1,34 @@
 
+To enable data validation for an editor, you need to declare the [Validator](/Documentation/ApiReference/UI_Components/dxValidator/) component and implement [validation rules](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/). You can attach multiple validation rules to one component. The following list contains all available validation rule types:
 
+- [required](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RequiredRule/)    
+A validation rule that demands that the validated field has a value.
 
-In&nbsp;the ASP.NET MVC&nbsp;demo, editors extract validation rules from data annotations attributed to&nbsp;the fields of&nbsp;the model (see the _EditorsViewModel.cs_ file).
+- [email](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/EmailRule/)    
+A validation rule that demands that the validated field matches the Email pattern.
 
+- [async](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/AsyncRule/)    
+A custom validation rule used for server-side validation. Implement the [validationCallback](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/AsyncRule/#validationCallback) function to validate the target value.
 
+- [compare](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/CompareRule/)    
+A validation rule that demands that the validated editor's value is equal to a specified expression. For this rule type, implement the [comparisonTarget](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/CompareRule/#comparisonTarget) function to specify the value that component compares to the validated value.
 
-Note that the _&laquo;Register&raquo;_ button here does not implement a&nbsp;usual **onClick** event handler. Instead, it&nbsp;has the **useSubmitBehavior** property set to _true_. This setting tells the button to&nbsp;validate and submit the HTML form in&nbsp;which it&nbsp;is&nbsp;nested, with no&nbsp;further configuration required.
\ No newline at end of file
+- [pattern](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/PatternRule/)    
+A validation rule that demands that the validated field matches a specified [pattern](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/PatternRule/#pattern). 
+
+- [stringLength](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/StringLengthRule/)   
+A validation rule that demands that the target value length is within the range between the specified [minimum](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/StringLengthRule/#min) and [maximum](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/StringLengthRule/#max) values. This property accepts only string values.
```

Comment:
```suggestion
A validation rule that requires the target value length to fall within the range of the specified [minimum](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/StringLengthRule/#min) and [maximum](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/StringLengthRule/#max) values. This property only accepts string values.
```
Please apply this edit to similar sentences in this doc.

Reply:
Okay